pymuonsuite/test/test_data/Si2/dftb-results/dftb+/Si2_1/geo_end.gen

```diff
@@ -0,0 +1,9 @@
+ 3  S
```

Comment:
Are all these `geo_end.gen` files needed for some test, or are they just leftovers? In case, let's remove them and add them to `.gitignore`.

Reply:
Currently, they are needed, because to read the dftb+ results you need to read the .gen file.

In `test_muairss.py`, `test_dftb` tests using muairss with dftb+. By default this test assumes dftb+ is not installed, so to be able to test the clustering with muairss with dftb+ results, we have to have the results saved, and this includes the `geo_end.gen` files.

Reply:
Ok, I just identified a few that can be deleted, so there are less now.